stage1/init/kvm/network.go

```diff
@@ -19,6 +19,7 @@ import (
 	"errors"
 	"fmt"
 	"io/ioutil"
+	"log"
```

Comment:
```
/tmp/go-build872694491/github.com/coreos/rkt/stage1/init/kvm/_test/_obj_test/network.go:24: log redeclared as imported package name
	previous declaration at $WORK/github.com/coreos/rkt/stage1/init/kvm/_test/_obj_test/log.go:25
```

The name conflict with `log` in `stage1/init/kvm/log.go`:
```
var log *rktlog.Logger
```

/cc @blixtra should that "log" variable be renamed to something else?